arches/management/commands/updateproject.py

```diff
@@ -31,6 +32,29 @@ def handle(self, *args, **options):
             self.stdout.write("Operation aborted.")
 
     def update_to_v8(self):
+        # Update apps.py to remove generate_frontend_configuration
```

Comment:
Maybe this is a little presumptive, but it seems unlikely projects have _innovated_ all that much with their apps.py